app/api/endpoints/auth.py

```
@@ -0,0 +1,59 @@
+
+
+@router.post(
+    "/device-token",
+    response_model=DeviceTokenUpdateResponse,
+    summary="디바이스 토큰 등록/갱신 (user_id 기반)",
+    description="클라 onNewToken에서 호출 — FCM/APNs 토큰이 갱신되면 서버에 다시 등록한다. "
+                "(최초 토큰은 카카오 로그인 시 device_token으로 함께 전송됨)",
+)
+async def register_device_token(
+    request: DeviceTokenRegisterRequest,
+    db: Session = Depends(get_db),
```

Ruff (0.15.18)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)